test/imports/LunaNext/Compositor/Singletons/WindowModelSingleton.qml

```diff
@@ -55,7 +55,7 @@ Item {
             else if( window.windowType === 6 )
                 windowModelSingleton.appendValue(pinListModel, {"window": window});
         }
-        onWindowRemovedFromListModel: {
+        function onWindowRemovedFromListModel() {
```

Comment:
Same here, it's `signal windowRemovedFromListModel(Item window);`
so you need function `onWindowRemovedFromListModel(window)`